docs/source/reference/pysupported.rst

```diff
@@ -514,9 +517,10 @@ explicitly from the `numba.typed` module::
 
 
 .. note::
-    As the typed-list stabilizes it will fully replace the reflected-list and the
-    constructors `[]` and `list()` will create a typed-list instead of a
-    reflected one.
+    The typed-list is beginning to stabilize with Numba version 0.47.0. Using
```

Comment:
Depends, it is still missing a few dunder methods and might have a refcount bug in `pop`. However, it hasn't significantly caught fire or caused a cascade failure anywhere we know of, so it will probably be alright. And if push comes to shove we can always hotfix and release a patch version.

Reply:
So my answer is: yes we should consider it stable.